### PR TITLE
Angularjs

### PR DESCRIPTION
**Pull request by Pristavka**

### CODE REVIEW

File: angularjs/app/components/header/header.component.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+(() => {
+  angular.module('headerModule').component('headerApp', {
+    templateUrl: 'components/header/header.html',
+    controller: 'Header',
+    controllerAs: 'ctrl'
```
Comment (NikitaSedyx) [inline angularjs/app/components/header/header.component.js]: it's better to use more unique names

File: angularjs/app/components/task-form/task-form.controller.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+(() => {
+  angular.module('taskFormModule').controller('TaskForm', TaskForm);
+
+  TaskForm.$inject = ['model', 'todoService'];
+  function TaskForm(model, todoService) {
+    this.todo = model;
+    this.disableAddButton = false;
+    Object.assign(this, todoService);
```
Comment (NikitaSedyx) [inline angularjs/app/components/task-form/task-form.controller.js]: there is practice to return this

File: angularjs/app/components/header/header.component.js (hunk 1 above)
Comment (Pristavka) [reply angularjs/app/components/header/header.component.js]: changed Header to HeaderController

File: angularjs/app/components/task-form/task-form.controller.js (hunk 2 above)
Comment (Pristavka) [reply angularjs/app/components/task-form/task-form.controller.js]: changed

**Pull request merged**